apps/laboratory/src/utils/ConstantsUtil.ts

```diff
@@ -58,22 +59,27 @@ const EvmNetworks = [
   zkSync,
   arbitrum,
   base,
+  baseSepolia,
   sepolia,
-  gnosis,
-  baseSepolia
-]
+  gnosis
+] as [AppKitNetwork, ...AppKitNetwork[]]
```

Comment:
Can we do `as const` ?

Reply:
The `createAppKit`'s `networks` parameter accepts `[AppKitNetwork, ...AppKitNetwork[]]`. IMO it's better to make sure it's happy with the type definitions.